lib/mix/lib/mix.ex

```diff
@@ -172,21 +172,21 @@ defmodule Mix do
       echo "world!"
 
       # mix.exs
-      defp create_aliases do
+      defp aliases do
         [
-          "taskalias": ["hex.info", "run priv/hello.exs", "cmd priv/world.sh"],
-          "taskalias2": ["run priv/hello1.exs", "run priv/hello2.exs"]
+          "task_alias": ["hex.info", "run priv/hello.exs", "cmd priv/world.sh"],
+          "task_alias2": ["run priv/hello1.exs", "run priv/hello2.exs"]
         ]
       end
 
-  In the example above we have created 2 aliases, the first example
-  `taskalias` will run task `hex.info`, then (`run`)[`Mix.Tasks.Run`]
+  In the example above, we have created two aliases. In the first example,
+  `task_alias` will run task `hex.info`, then (`run`)[`Mix.Tasks.Run`]
```

Comment:
Links in Markdown should be in the form `[]()` and not `()[]`. Do you mind changing that?

Reply:
fixed